Pod/Classes/PINAnimatedImageManager.m

```diff
@@ -39,15 +39,14 @@ + (instancetype)sharedManager;
 
 @end
 
-static dispatch_once_t startupCleanupOnce;
-
 @implementation PINAnimatedImageManager
 
 + (void)initialize
 {
   if (self == [PINAnimatedImageManager class]) {
-    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
-      dispatch_once(&startupCleanupOnce, ^{
+    static dispatch_once_t startupCleanupOnce;
+    dispatch_once(&startupCleanupOnce, ^{
+      dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
Wait! This was already running on an arbitrary queue :( Sorry, I misunderstood the previous code!